templates/templates/_navigation-developer-h.html

```diff
@@ -190,8 +190,8 @@ <h4 class="p-muted-heading">
           </div>
         </div>
         <div class="col-3">
-          <h4 class="p-muted-heading">Tutorials</h4>
-          <p class="p-p--small">Fun <a href="http://tutorials.ubuntu.com/">guides and HOWTOs</a> covering Ubuntu cloud, desktop, server, community, IoT, packaging and more.</p>
+          <h4 class="p-muted-heading"><a href="http://tutorials.ubuntu.com/">Tutorials</a></h4>
```

Comment:
can it have a &nbsp;&rsaquo; and https please... also, did you update the copy doc?